build/icons.json

```diff
@@ -0,0 +1,683 @@
+[
+  {
```

Comment:
I was envisioning something more like this:

```js
{
    "IconName": "<svg>...</svg>",
    "IconName2": "<svg>...</svg>"
}
```

The file as-is is mostly useless to me from a programming standpoint. Even if you didn't have the svg text and hardcoded the value to `1`, it'd be a marked improvement.